meterfs/meterfs.c

```diff
@@ -708,17 +715,44 @@ static int meterfs_writeRecord(file_t *f, const void *buff, size_t bufflen, mete
 	entry_t e;
 	unsigned int offset;
 	ssize_t wrote = 0, stat = 0;
+	void *tdata = NULL;
+	const void *data = buff;
+	uint8_t iv[32];
```

Comment:
Why is the IV 32 bytes? AES has a block size of 16 bytes. TinyAES in the function [AES_init_ctx_iv](https://github.com/kokke/tiny-AES-c/blob/f06ac37fc31dfdaca2e0d9bec83f90d5663c319b/aes.c#L224) simply ignores the rest if there are more than 16 bytes.

Reply:
future-proofing, this should allow seamless switch to AES256, should it be needed

Reply:
AES-256 has a *key size* of 32 bytes, but block size is still 16 bytes.

Reply:
Oh, you're right, thanks, will change that